src/window.cpp

```diff
@@ -3164,7 +3166,11 @@ void UpdateWindows()
 		/* Update viewport only if window is not shaded. */
 		if (w->viewport != NULL && !w->IsShaded()) UpdateViewportPosition(w);
 	}
+
+#ifdef ENABLE_NETWORKING
```

Comment:
This one is not needed, NetworkDrawChatMessage() is an static empty function without network